### DISCUSSION TITLE
Select all child nodes

### DISCUSSION THREAD

**Discussion by Antonimo-Dev** [freeplane/freeplane#1749] [Help]
Hi,

I'm sure I used to use Shift+Right and Shift+End to select all the nodes below the selected node. This behaviour does not seem to work now and I am wondering if this type of selection is still possible. If so, where can I go to find/change the key combination for this? I tried looking in Help » Command Search, but I did not find the answer.

Thanks,

**Comment by euu2021** [comment]
see https://github.com/freeplane/freeplane/discussions/677#discussioncomment-3729647

**Comment by Antonimo-Dev** [comment]
Thank you euu2021

After some trial and error I discovered that Ctrl+Shift+A selects all child nodes.

**Reply by euu2021** [reply]
Oh, I see. I thought that by "select all the nodes below the selected node" you meant all **siblings** below the selected node 🤣
